src/vcSettings.h

```diff
@@ -205,6 +206,13 @@ struct vcToolSettings
   } label;
 };
 
+struct HistoryProjectInfo
+{
+  const char *pName;
+  const char *pPath;
+  double date; // TODO:
```

Comment:
Why is this a `double`? Probably best to just not have this at all until it's required.